shenyu-bootstrap/src/main/resources/application.yml

```diff
@@ -86,6 +86,39 @@ shenyu:
         writeSpinCount: 16
         autoRead: true
         allocType: "pooled"
+  #  httpclient:
+  #    strategy: webClient
+  #    connectTimeout: 45000
+  #    responseTimeout: 3000
+  #    readerIdleTime: 3000
+  #    writerIdleTime: 3000
+  #    allIdleTime: 3000
+  #    readTimeout: 3000
+  #    writeTimeout: 3000
+  #    wiretap: false
+  #    keepAlive: false
+  #    pool:
+  #      type: ELASTIC
+  #      name: proxy
+  #      maxConnections: 16
+  #      acquireTimeout: 45000
```

Comment:
maybe need 
/**
         * Time in millis after which the channel will be closed,
         * if NULL there is no max idle time.
         */
        private Long maxIdleTime;

Reply:
ok